frontend/pulse3d/src/pages/uploads.js

```diff
@@ -156,6 +144,46 @@ export default function Uploads() {
   const [filtercolumn, setFilterColumn] = useState("");
   const [updateData, toggleUpdateData] = useState(false);
 
+  const uploadTableColumns = [
+    {
+      name: "Select",
+      center: false,
+      admin: false,
+      selector: (row) => (
+        <Checkbox id={row.id} checked={checkedUploads.includes(row.id)} onChange={handleCheckedUploads} />
+      ),
+    },
+    {
+      name: "File Owner",
+      admin: true,
+      center: false,
+      selector: (row) => row.username,
+    },
+    {
+      name: "Recording Name",
+      admin: false,
+      center: false,
+      selector: (row) => row.name || "none",
+    },
+    {
+      name: "Upload ID",
+      admin: false,
+      center: false,
+      selector: (row) => row.id,
+    },
+    {
+      name: "Created Date",
+      admin: false,
+      center: false,
+      selector: (row) => row.createdAt,
+    },
+    {
+      name: "Last Analyzed",
+      center: false,
+      admin: false,
+      selector: (row) => row.lastAnalyzed,
+    },
+  ];
   useEffect(() => {
     setTimeout(async () => {
```

Comment:
did you confirm that this setTimeout works and updates the job statuses without having to refresh the page?

Reply:
I have confirmed that it does send the request.